### ISSUE TITLE
Feeds are updated every tick, not just on schedule

### ISSUE THREAD

**Issue by dhasenan** [dhasenan/pierce#1]
```
bitnami@cloud /v/p/p/bin> sudo grep 'Garbage.*Evan Todd' /tmp/pierce2.log
2015-09-30 09:22:50,392 INFO pierce.FeedMaintenance.ExecuteSingle(:0) - running task pierce.GarbageCollector on feed [Feed: Id=558d746070313f201cb568b6, Uri=http://et1337.com/feed/, Title=Evan Todd]
2015-09-30 09:23:59,013 INFO pierce.FeedMaintenance.ExecuteSingle(:0) - running task pierce.GarbageCollector on feed [Feed: Id=558d746070313f201cb568b6, Uri=http://et1337.com/feed/, Title=Evan Todd]
2015-09-30 09:26:45,564 INFO pierce.FeedMaintenance.ExecuteSingle(:0) - running task pierce.GarbageCollector on feed [Feed: Id=558d746070313f201cb568b6, Uri=http://et1337.com/feed/, Title=Evan Todd]
2015-09-30 09:29:36,308 INFO pierce.FeedMaintenance.ExecuteSingle(:0) - running task pierce.GarbageCollector on feed [Feed: Id=558d746070313f201cb568b6, Uri=http://et1337.com/feed/, Title=Evan Todd]
2015-09-30 09:34:05,247 INFO pierce.FeedMaintenance.ExecuteSingle(:0) - running task pierce.GarbageCollector on feed [Feed: Id=558d746070313f201cb568b6, Uri=http://et1337.com/feed/, Title=Evan Todd]
2015-09-30 09:38:47,887 INFO pierce.FeedMaintenance.ExecuteSingle(:0) - running task pierce.GarbageCollector on feed [Feed: Id=558d746070313f201cb568b6, Uri=http://et1337.com/feed/, Title=Evan Todd]
2015-09-30 09:39:25,726 INFO pierce.FeedMaintenance.ExecuteSingle(:0) - running task pierce.GarbageCollector on feed [Feed: Id=558d746070313f201cb568b6, Uri=http://et1337.com/feed/, Title=Evan Todd]
2015-09-30 09:40:34,232 INFO pierce.FeedMaintenance.ExecuteSingle(:0) - running task pierce.GarbageCollector on feed [Feed: Id=558d746070313f201cb568b6, Uri=http://et1337.com/feed/, Title=Evan Todd]
2015-09-30 09:41:42,727 INFO pierce.FeedMaintenance.ExecuteSingle(:0) - running task pierce.GarbageCollector on feed [Feed: Id=558d746070313f201cb568b6, Uri=http://et1337.com/feed/, Title=Evan Todd]
2015-09-30 09:42:49,718 INFO pierce.FeedMaintenance.ExecuteSingle(:0) - running task pierce.GarbageCollector on feed [Feed: Id=558d746070313f201cb568b6, Uri=http://et1337.com/feed/, Title=Evan Todd]
2015-09-30 09:43:57,830 INFO pierce.FeedMaintenance.ExecuteSingle(:0) - running task pierce.GarbageCollector on feed [Feed: Id=558d746070313f201cb568b6, Uri=http://et1337.com/feed/, Title=Evan Todd]

```

**Comment by dhasenan** [comment]
Actually, that feed isn't showing up in my feed list. Curiouser and curiouser.

**Comment by dhasenan** [comment]
DMFA is also showing up every minute and is in my list. As is Skin Deep.

**Comment by dhasenan** [comment]
Okay, looks like there are two issues:
- We're mostly ignoring the minimum update interval from PierceConfig.
- We've got a bunch of subscriptions with their minimum update interval set to 0.

In GarbageCollector, we set the interval to the minimum of the intervals of the subscriptions.

Anyway, fixed in 2a45506.
